### PR TITLE
Migrated toggle-button to CSS modules

### LINKED ISSUE #571: Convert Toggle Button component to CSS modules
# Issue Description
To better align with modern CSS practices and modern CSS theming capabilities, we want to update this component to use CSS modules and if applicable, CSS custom properties.

## Issue Type

- [ ] New Feature
- [x] Enhancement
- [ ] Bug
- [ ] Other

## Expected Behavior
Component styles are converted to use CSS modules. If applicable, convert SCSS variables to CSS custom properties.

## Current Behavior
Component styles use manual name spaced classes and SCSS variables.

### PR DESCRIPTION
**Pull request by bjankord**
### Summary
Migrated toggle-button to CSS modules

@cerner/terra

Resolves #571 

### Deployment Link:
https://terra-core-deployed-pr-588.herokuapp.com/static/#/site/toggle-button

### CODE REVIEW

File: packages/terra-toggle-button/src/ToggleButton.jsx [hunk 1]
```diff
@@ -130,7 +132,7 @@ class ToggleButton extends React.Component {
           aria-label={closedButtonText}
           onClick={this.handleOnClick}
         >
-          <span className="terra-ToggleButton-icon">{icon}</span>
+          <span className={styles['button-text']}>{icon}</span>
```
Comment (mhemesath) [inline packages/terra-toggle-button/src/ToggleButton.jsx]: for consistency, I'd just use `cx` everywhere. e.g.
```
<span className={cx('button-text')}>{icon}</span>
```

Comment (bjankord) [inline packages/terra-toggle-button/src/ToggleButton.jsx]: Updated: https://github.com/cerner/terra-core/pull/588/commits/1cb65d35cae0b353f67e9bc807baaade3063ae84

File: packages/terra-toggle-button/src/ToggleButton.scss [hunk 2]
```diff
@@ -1,27 +1,29 @@
-@import '~terra-mixins';
```
Comment (emilyrohrbough) [inline packages/terra-toggle-button/src/ToggleButton.scss]: Should we remove `terra-mixins` from the package dependencies now?

**Comment by mmalaker** [comment]
Functional verification completed. Toggle button examples display and function correctly after conversion to CSS modules.

File: packages/terra-toggle-button/src/ToggleButton.scss (hunk 2 above)
Comment (bjankord) [inline packages/terra-toggle-button/src/ToggleButton.scss]: I thought about it, I'm planning to leave as-is for now. After we get all the components converted over, we could go through and remove it. Note sure if there will be any use for terra-mixins after we convert things over.

**Pull request merged**